src/org.ts

```diff
@@ -27,6 +29,40 @@ function validateOrgData(
   return undefined;
 }
 
+async function removeLastRelatedMember(
+  orgName: string,
+  options: ServiceOptions,
+): Promise<Result<undefined, OrgProblemType>> {
+  const memberModel = getModelForClass(MemberModelSchema, {
+    existingConnection: options.dbConnection,
+  });
+
+  const memberCount = await memberModel.count({ orgName });
+  if (memberCount > 1) {
+    options.logger.info({ orgName }, 'Refused deletion - existing org members');
```

Comment:
This style isn't consistent with our logs. I'd suggest something like:

```suggestion
    options.logger.info({ orgName }, 'Refused org deletion because it contains multiple members');
```